modules/javafx.controls/src/main/java/javafx/scene/control/TabPane.java

```diff
@@ -851,16 +773,14 @@ private Tab findNearestAvailableTab(int tabIndex, boolean doSelect) {
     }
 
 
-    // TabDragPolicy //
-    private ObjectProperty<TabDragPolicy> tabDragPolicy;
-
     /**
      * The drag policy for the tabs. The policy can be changed dynamically.
      *
      * @defaultValue TabDragPolicy.FIXED
```

Comment:
I think that the 2nd sentence is redundant again. I would add an explanation like "Specifies if tabs can be reordered or not" to elaborate on what a "drag policy" is.

Also, `{@code TabDragPolicy.FIXED}`.

Reply:
Fixed.